app/workers/sentry_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SentryJob
+  include Sidekiq::Worker
+
+  sidekiq_options queue: 'tasker'
```

Comment:
What queue should these be placed in?  I chose `tasker` because it's a lower priority. 
I thought about creating a new queue, but where is the Sidekiq config?

Reply:
I don't have a strong opinion here on which queue this would fall into. We only have one worker type that's not using default, and it's using critical. I would definitely not call this a low priority thing, but it would be isolated on the `tasker` queue if a backup did develop.